frontend/src/pages/DataCatalog/MethodologyTab.tsx

```diff
@@ -211,27 +205,31 @@ function MethodologyTab() {
                     of occurrence of COVID-19 cases expressed per 100,000 people
                     (i.e. 10,000 per 100K implies a 10% occurrence rate). This
                     metric normalizes for population size, allowing for
-                    comparisons across demographic groups.
+                    comparisons across demographic groups. This metric is
+                    rounded to the nearest integer in the tracker.
                   </li>
                   <li>
                     <b>
-                      Share of Total COVID-19 cases with Unknown Race and
-                      Ethnicity
+                      Share of total COVID-19 cases with unknown race and
+                      ethnicity
                     </b>
                     : Within a locale, the percentage of COVID-19 cases that
                     reported unknown race/ethnicity. For example, a value of 20%
                     for Georgia means that 20% of Georgia's reported cases had
-                    unknown race/ethnicity.
+                    unknown race/ethnicity. This metric is rounded to two
+                    decimal places in the tracker.
```

Comment:
The rounding part is not technically correct. I'd replace:
`This metric is rounded to two decimal places in the tracker.`
with
`This metric is rounded to one decimal place. In instances where this would round to 0%, two decimal places are used.`

Same below

Reply:
Yeah, I was debating whether to include this detail or not, sg to add it